src/theme/components/tooltip.ts

```diff
@@ -26,7 +27,7 @@ export const Tooltip = defineStyleConfig({
       )(props),
       border: '1px solid',
       borderColor: mode('primary-light.500', 'primary-dark.600')(props),
-      borderRadius: 'md',
+      borderRadiusStandard
```

Comment:
This isn't going to work

Reply:
I have a feeling the code line referenced is outdated, the current tweak is 'md' which corresponds to the 6px. Seems to be working from what I tried a few minutes ago